monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/micro/integrator/commons/DelegatesUtil.java

```diff
@@ -76,8 +98,80 @@ public static Artifacts getArtifactsFromMI(String groupId, List<String> nodeList
                 }
             }
         }
+        
+        //ordering   
+        Comparator<ArtifactsInner> comparatorObject;
+        switch (orderBy.toLowerCase()) {
+            //for any other ordering options
+            default: comparatorObject = Comparator.comparing(ArtifactsInner::getNameIgnoreCase); break;
+        }
+        if (order.equalsIgnoreCase("desc")) {
```

Comment:
Can order be null? Shall we switch the two params to make sure there won't be NPE.

Reply:
from the UI it sends a default value for both order and orderBy. But if something happened  while sending the data through http, in between the web UI and core, it can be null. Otherwise not